src/plugins/curator/components/widget-screen/DataSourceTab.js

```diff
@@ -65,7 +74,7 @@ export default function DataSourceTab(props) {
             name="search_request._schema"
             label="Content Type"
             options={options}
-            isClearable={false}
+            isClearable={true}
```

Comment:
@rsumilang nit: should be able to to just `isClearable` with no value and it will be true